cukinia/common_security_tests.d/auditd.conf

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
+cukinia_log "$(_colorize yellow "--- check auditd configuration ---")"
+modules_function=" \
+    insmod \
+    kmod \
+    modprobe \
+    rmmod \
+"
+
+syscall_list=" \
+    ioperm \
+    prctl \
+    ptrace \
+"
+
+file_monitoring="\
+    open \
+    openat \
+    unlink \
+"
+
+as "ANSSI-BP-028-R73 - auditd service is active" \
+    cukinia_systemd_unit auditd
+
+as "ANSSI-BP-028-R73 - auditd is configure to output in syslog" cukinia_cmd \
+    grep -q "active = yes" /etc/audit/plugins.d/syslog.conf
+
+
+for function in ${modules_function}; do
+    /sbin/$function &>/dev/null
+    as "ANSSI-BP-028-R73 - $function call is log" cukinia_cmd \
+        grep -q $function /var/log/audit/audit.log
+done
+
+touch /etc/test
```

Comment:
/etc/test is a very generic name that might collide with something else. `touch /etc/cukinia-auditd-test`